docs/getting-started/my-account/README.md

```diff
@@ -61,15 +61,15 @@ Also, you can activate a **warning if you are leaving a work package with unsave
 
 Additionally, you can activate to **auto-hide success notifications** from the system. This (only) means that the green pop-up success notifications will be removed automatically after five seconds.
 
-![profile-settings](profile-settings.png)
+![my-account-settings](profile-settings.png)
 
 ### Change your language
 
 To change the language in OpenProject, navigate to the **My account** settings and choose the menu point **Settings**.
 
 Here you can choose between multiple languages.
 
-![select a language](1572882728878.png)
+![my-account-change-language](Bildschirmfoto 2022-03-02 um 14.44.45.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.